adapter/memquota2/dedup.go

```diff
@@ -105,7 +105,7 @@ func (du *dedupUtil) reapDedup() {
 	du.oldDedup = du.recentDedup
 	du.recentDedup = t
 
-	if du.logger.VerbosityLevel(4) {
+	if du.logger.VerbosityLevel(6) {
 		du.logger.Infof("Running repear to reclaim %d old deduplication entries", len(t))
```

Comment:
we might want to update "repear" to be `reaper`, while we are at it.

Reply:
Can we not log anything when len(t) is 0 ?